code/frontend/src/app/_directives/alert.component.ts

```diff
@@ -39,8 +38,7 @@ export class AlertComponent implements OnDestroy {
         if (message !== undefined) {
 
             console.log("growl: "+JSON.stringify(message)  );
-            this.growlMsgs.push(message);
-            this.lastMessage = [];
```

Comment:
tutaj bylo wyzerowanie tablicy, na pewno nie jest juz potrzebne?

Reply:
Był problem bo jak się pojawia błąd to są dwie wiadomości, jedna z backend druga z alertServcie. Jak było czyszczenie to nie zapisywał poprzedniej wiadomości i wyświetlała się tylko ostatnia. Czyszczenie tablicy jest też w metodzie clean. Dodałem także czyszczenie w else w tej metodzie bo ona się uruchamia po akcji w interfejsie i jako parametr jest undefined i wtedy zeruje tablicę.

Reply:
OK,